kornia/geometry/nerf/nerfmm/utils/lie_group_helper.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import torch
+from scipy.spatial.transform import Rotation as RotLib
+
+
+def SO3_to_quat(R):
```

Comment:
check on the conversions module because we have some of this functionalities
https://kornia.readthedocs.io/en/latest/geometry.conversions.html

Reply:
in case of missing we can implement there